### PR TITLE
Add frontend of receive screen

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot from 2023-06-06 10-03-36

### CODE REVIEW

**Review by Pierregvx — APPROVED** [review]

File: web-frontend/src/pages/TransferCoins/ReceiveCoins/CopyAddress.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Button, Input } from '@massalabs/react-ui-kit';
+import { FiCopy, FiLink } from 'react-icons/fi';
+
+function CopyAddress() {
+  return (
+    <div className="flex flex-col h-full items-center">
+      <p>Accout Address:</p>
+      <Button posIcon={<FiCopy />}> Generic Address </Button>
```
Comment (mazmassa) [inline web-frontend/src/pages/TransferCoins/ReceiveCoins/CopyAddress.tsx:8]:
```suggestion
      <Button posIcon={<FiCopy />}>Generic Address</Button>
```

File: web-frontend/src/pages/TransferCoins/ReceiveCoins/ReceiveCoins.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import CopyAddress from './CopyAddress';
+
+function ReceiveCoins() {
+  return (
+    <WalletLayout menuItem={MenuItem.SendCoins}>
+      <div className="flex flex-row items-center gap-5">
+        <img src="https://placehold.jp/165x165.png"></img>
+        <CopyAddress />
+      </div>
+      ;
```
Comment (mazmassa) [inline web-frontend/src/pages/TransferCoins/ReceiveCoins/ReceiveCoins.tsx:13]: 🤔

File: web-frontend/src/pages/TransferCoins/ReceiveCoins/CopyAddress.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      <Button preIcon={<FiLink />}>Generate a link</Button>
```
Comment (mazmassa) [inline web-frontend/src/pages/TransferCoins/ReceiveCoins/CopyAddress.tsx:9]: Why not translations keys!?

**Review by mazmassa — APPROVED** [review]

**Review by mazmassa — APPROVED** [review]

File: web-frontend/src/i18n/en_US.json [hunk 4]
```diff
@@ -89,5 +89,9 @@
     "custom-fees": "Custom fees : ",
     "confirm-fees": "Confirm fees ",
     "gas-info": "Default gas fees are XX and defined by Massa Labs to assure minimum tip to block validator. If you want to custom them go to advanced section in previous screen"
+  },
+  "receive": {
+    "account-receive": "Generate a link",
+    "account-address": "Account address : "
```
Comment (Thykof) [inline web-frontend/src/i18n/en_US.json:95]:
```suggestion
    "account-address": "Account address: "
```

File: web-frontend/src/pages/TransferCoins/ReceiveCoins/GenerateLink.tsx
Comment (Thykof) [inline web-frontend/src/pages/TransferCoins/ReceiveCoins/GenerateLink.tsx:1]: what is the purpose of this empty file?

Comment (Thykof) [reply web-frontend/src/pages/TransferCoins/ReceiveCoins/GenerateLink.tsx:1]: okay, in that case adding an empty file could have been done in the following PR. always think that this PR could go in prod and we don't want empty file in prod codebase.

**Pull request merged**